### PR TITLE
Remove react scripts ts

### LINKED ISSUE #1: Replace react-scripts-ts
This project currently uses [react-scripts-ts](https://github.com/wmonk/create-react-app-typescript), which has been deprecated. Create react app now natively supports typescript and should be used from here on out.

Here is a helpful guide for migrating from react-scripts-ts to react-scripts: https://vincenttunru.com/migrate-create-react-app-typescript-to-create-react-app/

### PR DESCRIPTION
**Pull request by jpreese**
Conversion from react-scripts-ts to react-scripts.

This should resolve #1

### CODE REVIEW

**Comment by numo16** [comment]
Looks like your package-lock.json is referencing a private repository, following the removing and adding of various packages. We could resolve this by having a [project level .npmrc file](https://docs.npmjs.com/files/npmrc#per-project-config-file) that explicitly uses `registry="https://registry.npmjs.org/"`.

Additionally, related to the topic of package management, we should probably not be mixing yarn and npm for package management in this repo. Standardizing on just npm and ditching the yarn.lock file in this repo probably makes the most sense.

**Comment by jpreese** [comment]
Good catch! Added npmrc to prevent this from occurring in the future and updated the package-lock.

I would personally prefer to only use npm. I'll open an issue.

File: .npmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+registry="https://registry.npmjs.org/"
```
Comment (numo16) [inline .npmrc:1]: ignore previous comment, this should be fine

**Comment by numo16** [comment]
I'm encountering something odd after these changes. When running the app locally, in chrome 72.0.3626.96 or firefox 65.0.1, the definitions editor seems to be rendering incorrectly. The contents of the editor are blank (no line numbers, no default text, no scroll bars). Here is a screenshot of what I'm seeing in both browsers:

image

I checked in react devtools and the value is definitely being set on the component, so it could be that whatever happened in switching over to react-scripts is changing how the css is being bundled?

image

**Comment by jpreese** [comment]
Similar to https://github.com/graphql/graphiql/issues/33 , this does seem to be a loading issue. If I call codemirror.css directly from public/index.html, rather than an import, the issue does go away.

**Comment by numo16** [comment]
Noticing that vendoring in public fixes it. It is also called out in [create react apps's](https://facebook.github.io/create-react-app/docs/using-the-public-folder) docs as an escape hatch for things that need to be loaded explicitly. The downside to doing it that way is that we have to be conscious of that stylesheet being vendored in the public folder if we ever upgrade the codemirror package.

**Comment by jpreese** [comment]
Yeah, I moved codemirror from node_modules to the public dir to force it to load ahead of the app, but it felt a little wrong for the reason you mentioned. Though it's currently the only solution I've found.

**Comment by jpreese** [comment]
Moved the codemirror stylesheet to the public directory

**Comment by numo16** [comment]
Everything is working as expected now and takes care of the referenced issue. Update the changelog (since dependency loading was changed a bit) and we're good to go.

**Comment by numo16** [comment]
Regarding versioning in the changelog, semantic versioning doesn't tend to fit well with applications. It would be best to move to just updating the changelog with releases by date.

**Review by numo16 — APPROVED** [review]
Everything looks good.

**Pull request merged**